docs/advisors/checks/postgresql-eol-check.md

```diff
@@ -0,0 +1,17 @@
+# Percona Distribution for PostgreSQL End Of Life
+## Description
+Percona Distribution for PostgreSQL, you are using reached End-of-life.
+Percona does not provide support or issue hotfixes for end-of-life releases.
+
+After that date, the software will be out of support, and users that will still be running that version will not
+get any further updates or bug fixes.
```

Comment:
I think the sentence above covers what we mean to say here. I would remove this sentence and not the one above because we don't actually mention any date.